elements/rh-site-status/test/rh-site-status.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { expect, html } from '@open-wc/testing';
+import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+import { RhSiteStatus } from '@rhds/elements/rh-site-status/rh-site-status.js';
+
+import { stub, type SinonStub } from 'sinon';
+
+describe('<rh-site-status>', function() {
+  let element: RhSiteStatus;
+  let fetchStub: SinonStub;
+
+  describe('simply instantiating', function() {
+    beforeEach(async function() {
+      fetchStub = stub(window, 'fetch');
+      fetchStub.resolves({
+        ok: true,
+        status: 200,
+        statusText: 'OK',
+        json: () => Promise.resolve({
+          status: {
+            indicator: 'none',
+            description: 'All systems operational'
+          }
+        })
+      });
+      element = await createFixture<RhSiteStatus>(html`<rh-site-status></rh-site-status>`);
+      await element.updateComplete;
+    });
+
+    afterEach(function() {
```

Comment:
you can put this in the same describe block as the declaration, then you don't need to remember to put it everywhere else

just use optional chaining to avoid errors for blocks that don't stub fetch
```js
    afterEach(function() {
      fetchStub?.restore?.();
    });